app/dashboard/templates/ens/base.html

```diff
@@ -0,0 +1,62 @@
+{% load static %}
+<!DOCTYPE HTML>
+<html>
+	<head>
```

Comment:
2 space indentation throughout

Reply:
this can be fixed programmatically with `make fix` right?

Reply:
It won't automatically format `.html` files at the moment.